client/my-sites/themes/themes-banner/index.jsx

```diff
@@ -5,31 +5,56 @@
  */
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import cssSafeUrl from 'lib/css-safe-url';
 import Button from 'components/button';
+import safeImageUrl from 'lib/safe-image-url';
 
 export class ThemesBanner extends PureComponent {
 	static propTypes = {
-		title: PropTypes.string.isRequired,
-		description: PropTypes.string.isRequired,
-		action: PropTypes.func,
+		title: PropTypes.node.isRequired,
+		description: PropTypes.node.isRequired,
+		action: PropTypes.func.isRequired,
 		actionLabel: PropTypes.string.isRequired,
-		backgroundImage: PropTypes.string,
-		href: PropTypes.string,
+		backgroundColor: PropTypes.string,
+		image: PropTypes.string,
+		imageAttrs: PropTypes.shape( {
+			alt: PropTypes.string,
+			width: PropTypes.number,
+		} ),
+		imageTransform: PropTypes.string,
 	};
 
 	render() {
-		const { title, description, actionLabel, action, backgroundImage, href } = this.props;
-		const backgroundStyle = backgroundImage
-			? { backgroundImage: `url( ${ cssSafeUrl( backgroundImage ) } )` }
-			: null;
+		const {
+			title,
+			description,
+			actionLabel,
+			action,
+			backgroundColor,
+			image,
+			imageAttrs = {},
+			imageTransform = 'auto',
+		} = this.props;
+		const backgroundStyle = backgroundColor ? { backgroundColor } : {};
+		const imageNode = image ? (
+			<img
+				alt=""
+				{ ...imageAttrs }
```

Comment:
Since it's not totally obvious that `...imageAttrs` contains an `alt` attr, we could add a `themeName` prop to `ThemesBanner` that we use to set `alt={ translate( '%(theme)s Theme, args: { theme: themeName } ) }`.